### ISSUE TITLE
Clarifications regarding Consent API and CDS release dates

### ISSUE THREAD

**Issue by dimuthnc** [ConsumerDataStandardsAustralia/infosec#68]
**Description**
In the 0.1 version of information security profile (Christmas draft release), it was mentioned that a Consent API will be defined ( in a later release) to **create, query and delete** consents. However, we came across with this comment[1] in an information security profile Github issue. This indicates that Consent API will not be defined for V1 release of CDS and consents will be handled via OIDC scopes. 

Can you confirm that this decision is finalized? 

Apart from that, as a company working on a reference implementation for CDS, we are keen to know the release dates of V1. Please let us know your plans regarding this. 

[1] - https://github.com/ConsumerDataStandardsAustralia/infosec/issues/47#issuecomment-479275972

**Comment by JamesMBligh** [comment]
Yes.  The consent API will not be included in the V1 specification and OIDC scopes and claims will be used for defining consent to share data.  The consent API may be introduced in a later version if low granularity permissions are required.

This is now reflected in [Decision Proposal 64](https://github.com/ConsumerDataStandardsAustralia/standards/issues/64).  Feedback on that decision proposal would be welcome.

As for dates, the intent is for version 1 of the API and InfoSec standards to be published at the end of May.

-JB-

**Comment by dimuthnc** [comment]
Hi James,

According to the previous comment on this issue, you were planning to release 1.0.0 of API and InfoSec standards by the end of May. We have noted that CDS [0.9.3 version](https://consumerdatastandardsaustralia.github.io/standards/#change-log) is published now with the description  “Final updates for May Draft”. Decision Proposal 70 and Decision Proposal 71 has created to collect feedback for the May 2019 Working draft and those will be open till **21st June**.

Since it is the end of May, please let us know your plans regarding release dates of the 1.0.0 version. Will this be released after merging the Decision proposal feedback ( 21st June)? 

Regards,
Dimuth
